addons/base_address_extended/models/res_partner.py

```diff
@@ -80,6 +80,9 @@ def _split_street_with_params(self, street_raw, street_format):
         vals = {}
         previous_pos = 0
         field_name = None
+        # remove whitespaces
+        street_format = street_format.strip()
```

Comment:
I don't understand why this would be necessary, the street_format that you define doesn't have a space before or after